pkg/cmd/roachtest/allocator.go

```diff
@@ -23,11 +23,9 @@ import (
 
 func registerAllocator(r *testRegistry) {
 	runAllocator := func(ctx context.Context, t *test, c *cluster, start int, maxStdDev float64) {
-		const fixturePath = `gs://cockroach-fixtures/workload/tpch/scalefactor=10/backup`
 		c.Put(ctx, cockroach, "./cockroach")
-		c.Put(ctx, workload, "./workload")
 
-		// Start the first `start` nodes and restore the fixture
+		// Start the first `start` nodes and restore a tpch fixture.
```

Comment:
I had trouble loading the fixture locally so I just went and switched this to import and also use the built-in workload binary instead of the external one.